README.md

```diff
@@ -51,6 +52,14 @@ $ ./tools/sof-dump-status.py -p
 apl
 ```
 
+### test case result
+| exit code | display    | discription                                        |
+| --------- | ---------- | -------------------------------------------------- |
+| 0         | PASS       | Test has passed                                    |
+| 1         | FAIL       | Test has failed                                    |
+| 2         | SKIP       | Test is not applicable for this target or topology |
```

Comment:
SKIP? or N/A

Reply:
ok, I will update for direct convert 'skip' to 'NA'